src/App.css

```diff
@@ -1,22 +1,3 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-}
-
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  font-size: calc(10px + 2vmin);
-  color: white;
-}
-
-.App-link {
-  color: #09d3ac;
+li {
+  color: red;
```

Comment:
Second thoughts: This file will definitely be useful after my other comments in pushing the nav links to the bottom of the page. Instead of using `absolute` positioning, trying looking into [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox), which will allow you to do some of the same things but in a more responsive (and modern) way. 😄

I realize I've commented a TON here and it probably seems like a lot of work. Give it a shot and reach out if you have any questions!

Reply:
Hi Kevin, I tried to disable the red "flash" by removing:
li {
  color: red;
}


from App.js.
Didn't seem to work for me though.

Reply:
I see, it's possible that it's coming from somewhere else. It was just an observation and really shouldn't hold up this pr. I'm about to review this changes, but I suspect that it won't be relevant anymore. No worries. 😄